compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -870,8 +893,8 @@ convertChoice env tbinds (ChoiceData ty expr)
         applyThisAndArg func = func `ETmApp` EVar this `ETmApp` EVar arg
 
 
-convertBind :: Env -> (Var, GHC.Expr Var) -> ConvertM [Definition]
-convertBind env (name, x)
+convertBind :: Env -> S.Set T.Text -> (Var, GHC.Expr Var) -> ConvertM [Definition]
+convertBind env interfaceCons (name, x)
```

Comment:
Instead of passing around `interfaceCons`, I think it would be nicer to have it in the `env`.

Reply:
good point, I orginally needed the `ConvertM` which made this a bit annoying to fit in but now that I don’t, this works nicely.